src/ert/__main__.py

```diff
@@ -26,6 +28,7 @@
 )
 from ert.cli.main import ErtCliError, ErtTimeoutError, run_cli
 from ert.config import ConfigValidationError, ErtConfig, lint_file
+from ert.gui.main import run_gui
```

Comment:
This will import `ert.gui` when cli is used. We should check that this 1. does not have unintended side-effects and 2. does not slow down startup time for cli.